extensions/wikia/EditAccount/SpecialEditAccount_body.php

```diff
@@ -308,7 +308,7 @@ function setPassword( $pass, $changeReason = '' ) {
 		try {
 			// wrap in try/catch in case of PasswordException
 
-			if ( $this->mUser->setPassword( $pass ) ) {
+			if ( $this->mUser->setPasswordAndClearTokens( $pass ) ) {
 				global $wgUser, $wgTitle;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgUser" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)